src/test/java/io/github/fablabsmc/fablabs/api/fluidvolume/v1/FractionTest.java

```diff
@@ -1,23 +1,24 @@
 package io.github.fablabsmc.fablabs.api.fluidvolume.v1;
 
-import static io.github.fablabsmc.fablabs.api.fluidvolume.v1.Fraction.ONE;
-import static io.github.fablabsmc.fablabs.api.fluidvolume.v1.Fraction.ZERO;
-import static io.github.fablabsmc.fablabs.api.fluidvolume.v1.Fraction.deserialize;
-import static io.github.fablabsmc.fablabs.api.fluidvolume.v1.Fraction.multiply;
-import static io.github.fablabsmc.fablabs.api.fluidvolume.v1.Fraction.of;
-import static io.github.fablabsmc.fablabs.api.fluidvolume.v1.Fraction.ofValidDenominator;
-import static io.github.fablabsmc.fablabs.api.fluidvolume.v1.Fraction.ofWhole;
-
 import com.google.gson.JsonArray;
 import com.google.gson.JsonPrimitive;
 import com.mojang.datafixers.Dynamic;
 import com.mojang.datafixers.types.JsonOps;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.math.Fraction;
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.ImmutableFluidVolume;
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.MultiFluidContainer;
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.api.FixedFractionFixedSizeFluidVolume;
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.api.FixedSizedFluidVolume;
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.api.FluidVolume;
+import net.minecraft.Bootstrap;
 import net.minecraft.datafixer.NbtOps;
+import net.minecraft.fluid.Fluids;
 import net.minecraft.nbt.IntArrayTag;
 import net.minecraft.util.Util;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static io.github.fablabsmc.fablabs.api.fluidvolume.v1.math.Fraction.*;
 
 public class FractionTest {
```

Comment:
A lot of the tests here seem to also test the volume impls. Could you split this up into multiple classes?